src/KristofferStrube.Blazor.FileSystemAccess/FileSystemAccessService.cs

```diff
@@ -151,6 +159,46 @@ public async Task<bool> IsSupportedAsync()
             await jSRuntime.InvokeAsync<bool>("window.hasOwnProperty", "showDirectoryPicker");
     }
 
+    #region Create Handle Instances
```

Comment:
For the following methods, we can use the jSRuntime member instead of taking it as an argument. This would be one of the pros of moving the creator methods to the service.